git/git.go

```diff
@@ -490,13 +490,45 @@ func (c *gitConfig) IsGitVersionAtLeast(ver string) bool {
 	return IsVersionAtLeast(gitver, ver)
 }
 
+// Prepend Git config instructions to disable Git LFS filter
+func PrependConfigToDisableLFS(args ...string) []string {
```

Comment:
Additionally, I'm not sure that disabling LFS in all uses of `Exec()`, and `SimpleExec()` is safe. If you want to keep this function, I'd recommend that we instead change the calls from:

```go
// Exec runs Git, disabling LFS, with the given arguments.

func Something() ([]byte, error) {
        return Exec("something", ...)
}
```

to:

```diff
@@ -1,5 +1,5 @@
-// Exec runs Git, disabling LFS, with the given arguments.
+// Exec runs Git with the given arguments.
 
func Something() ([]byte, error) {
-        return Exec("something", ...)
+        return Exec(PrependConfigToDisableLFS("something")...)
}
```